name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/ComdirectPDFExtractor.java

```diff
@@ -515,6 +527,31 @@ private void addDividendTransactionFromSteuermitteilungPDF()
                         
                         .wrap(TransactionItem::new));
     }
+    
+    @Override
+    public List<Item> removeDuplicateDividendsWithoutTax(List<Item> items)
+    {
+     
+        // group dividends into tax + nontax
+       Map<LocalDateTime, Map<Security, List<Item>>> dividends = items.stream()
+            .filter(i -> "Dividend".equals(i.getTypeInformation())) //$NON-NLS-1$
```

Comment:
Das funktioniert so nur, wenn man mit den englischen Locale arbeitet.

Ich denke Du musst schauen ob i vom Typ TransactionItem ist, dann checken ob die Transaction eine AccountTransaction ist und vom Typ AccountTransaction.Type.DIVIDEND

Reply:
das klingt weniger quick-and-dirty :-)